response_operations_ui/views/messages.py

```diff
@@ -532,6 +517,18 @@ def view_selected_survey(selected_survey):  # noqa: C901
 
         pagination = pagination_processor(tab_counts["current"], limit, page, href)
 
+        if thread_id:
+            thread_url = url_for(
+                "messages_bp.view_conversation",
+                thread_id=thread_id,
+                conversation_tab=conversation_tab,
+                page=page,
+                ru_ref_filter=ru_ref_filter,
+                business_id_filter=business_id_filter,
+            )
```

Comment:
This is not quite the right url its causing a different reaction from the site it needs the latest message bit added to the end 
```
thread_url = (
           url_for(
                    "messages_bp.view_conversation",
                    thread_id=thread_id,
                    page=page,
                    conversation_tab=conversation_tab,
                    ru_ref_filter=ru_ref_filter,
                    business_id_filter=business_id_filter,
                )
                + "#latest-message"
            )
```

Reply:
Fixed.